Cargo.toml

```diff
@@ -21,16 +21,25 @@ path = "bin/inx-chronicle/src/main.rs"
 [dependencies]
 async-recursion = { version = "1.0", default-features = false }
 async-trait = { version = "0.1", default-features = false }
-clap = { version = "3.1", default-features = false, features = [ "derive", "std" ] }
+clap = { version = "3.1", default-features = false, features = ["derive", "std"] }
 dotenv = { version = "0.15", default-features = false }
 env_logger = { version = "0.9", default-features = false, features = ["termcolor", "atty", "humantime"] }
 futures = { version = "0.3", default-features = false }
-inx = { git = "https://github.com/iotaledger/inx", version = "0.2", default-features = false }
+inx = { path = "../inx/rust", version = "0.2", default-features = false, features = ["types"] }
 log = { version = "0.4", default-features = false }
-mongodb = { version = "2.1", default-features = false, features = ["tokio-runtime"] }
+mongodb = { version = "2.1", default-features = false, features = ["bson-chrono-0_4", "tokio-runtime"] }
```

Comment:
The `bson-chrono-0_4` feature should not be needed here.

Reply:
You're right, thanks!